components/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/entitlement/XacmlEntitlementServiceImpl.java

```diff
@@ -187,12 +187,12 @@ public void generateAndSaveEntitlementPolicies(List<XACMLPolicyTemplateContext>
     }
 
     @Override
-    public boolean isPermitted(EntitlementDecisionRequest request) {
+    public boolean isPermitted(EntitlementDecisionRequest request) throws AppManagementException {
 
         PEPProxy pepProxy = getPepProxy();
 
         if(pepProxy == null){
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/207d3b1c14192fee91f6d02b39c54f42bdf46e92